Formula/abuse.rb

```diff
@@ -23,6 +23,10 @@ class Abuse < Formula
     sha256               x86_64_linux:   "99bb1d87aef39b46eb39d0b6d9483afa3052bb0e616a16bbea90798545f1db37"
   end
 
+  # Uses deprecated `sdl_mixer`. HEAD does have support for SDL 2 but wasn't released.
+  # Last release on 2011-05-09 and last SVN revision on 2014-07-21
+  deprecate! date: "2023-02-05", because: :unmaintained
```

Comment:
Last revision added SDL2 support but looks like a number of changes went in between this and 0.8 so may not be worth trying to backport: http://abuse.zoy.org/log/abuse/trunk?action=stop_on_copy&mode=stop_on_copy&rev=737&stop_rev=&limit=100

Some Linux distros (e.g. Gentoo) use fork like https://github.com/Xenoveritas/abuse but not official.

Reply:
The download count is pretty low for this formula. I don't think it's worth investing much energy into this. If upstream makes a new release we can reconsider this.